pkg/cmd/stack/translate.go

```diff
@@ -667,8 +667,8 @@ func translateResources(svc *model.Service) apiv1.ResourceRequirements {
 		if svc.Resources.Requests.Memory.Value.Cmp(resource.MustParse("0")) > 0 {
 			if result.Requests == nil {
 				result.Requests = apiv1.ResourceList{}
-				result.Requests[apiv1.ResourceMemory] = svc.Resources.Requests.Memory.Value
 			}
+			result.Requests[apiv1.ResourceMemory] = svc.Resources.Requests.Memory.Value
```

Comment:
Could we add a test for this function? These are the use cases that we need to test:
- `svc.Resources` not defined by the user
- `svc.Resources.Limits` defined:
  - `svc.Resources.Limits.CPU` defined
  - `svc.Resources.Limits.CPU` not defined
  - `svc.Resources.Limits.Memory` defined
  - `svc.Resources.Limits.Memory` not defined
- `svc.Resources.Requests` defined:
  - `svc.Resources.Requests.CPU` defined
  - `svc.Resources.Requests.CPU` not defined
  - `svc.Resources.Requests.Memory` defined
  - `svc.Resources.Requests.Memory` not defined

Reply:
:point_up: Yes please. Javi's description is great to do some table tests